dev/docker/3-ursulas.yml

```diff
@@ -1,61 +0,0 @@
-version: '3'
-
```

Comment:
Is 3-ursulas gone forever now? How much more intensive is it run run 8?

Reply:
I renamed this "8-federated-ursulas".  I don't think it is an issue to run 8 as opposed to 3.  I just figured it is more useful.  I was thinking we could add a few more scenarios like, 8-decentralized-and-a-felix (with a ganache), etc.  

nothing is forever!

Reply:
Of course - Keeping in mind future hackathons, I'm trying to keep track of the best way to use docker in local development, as well as review the production automation.  

Is this still the best way for local fleet development?

Reply:
yeah, well I would say as with anything people should look at the demos.
I changed it to 8 instead of 3 mostly for those hackathon situations just because it allows for more realistic m/n scenarios without having to make a whole additional file.  But I really am just guessing.  If you think 3 is better I'm fine with changing it back.

Reply:
Not a suggestion - just an inquiry.  Thanks for explaining it to me